docs/Zendesk-batchsource.md

```diff
@@ -21,13 +21,6 @@ Configuration
 
```

Comment:
Can you change the CDAP data types to lowercase?  string is lowercase but the others in the Data Type table are init cap, i.e. Boolean should be boolean.

Reply:
done